src/bin/cobalt/main.rs

```diff
@@ -227,7 +227,23 @@ fn run() -> Result<()> {
                                  .default_value("cobalt site import")
                                  .takes_value(true)))
         .subcommand(SubCommand::with_name("list-syntax-themes").about("list available themes"))
-        .subcommand(SubCommand::with_name("list-syntaxes").about("list supported syntaxes"));
+        .subcommand(SubCommand::with_name("list-syntaxes").about("list supported syntaxes"))
+        .subcommand(SubCommand::with_name("convert-jekyll")
+                        .about("convert jekyll website to cobalt")
+                        .arg(Arg::with_name("source")
```

Comment:
My guess is that the CI is failing because there is a global `source`/`destination` and a local one.

Not sure why you aren't running into it.

Reply:
I've runned clippy but nothing came out :-/ I'll change the name of the option

Reply:
Do I need to change the long/short names as well? Or only the `with_name` one?